app.js

```diff
@@ -18,40 +19,55 @@ go();
 async function go() {
   try {
     for (const org of config.orgs) {
+      console.log('⏳ Collecting repos for', org)
```

Comment:
Added these logs since it can take a while with no feedback. It also helps make it easier to see separation between commit lists.